certificate-renewal/REPWRITERSPECS/BANNO.CDRENEW.V1.POW

```diff
@@ -251,9 +248,57 @@ DEFINE
  SYMXCLIENTNUM             = NUMBER
  ALLOWINMEMOMODE           = NUMBER
  PROGRAMVERSION            = CHARACTER
-END [DEFINE]
+
+[* TKainz - added 01/31
+*]
+ BNOCFGFILENAME            = CHARACTER
+ BNODEBUGCOUNT             = NUMBER
+ BNODEBUGLOOP              = NUMBER
+ BNODEBUGMODE              = NUMBER
+ BNODEBUGMODEOFF           = 0
+ BNODEBUGMODEON            = 1
+ BNODEBUGMSG               = CHARACTER
+ BNODEBUGMSGLIST           = CHARACTER ARRAY(999)
+ BNODEBUGMSGMAX            = 999
+ BNOIDLENGTH               = NUMBER
+ BNOINDENT                 = CHARACTER ARRAY(9)
+ BNOINDENTMAX              = 9
+ BNOLASTMODDATE            = DATE
+ BNOLASTMODTIME            = CHARACTER
+ BNOLOCALLOOP              = NUMBER
+ BNOPRINTDEBUGDAYS         = NUMBER
+ BNOPRINTDEBUGINFODAYS     = NUMBER
+ BNOPROGRAMNAME            = CHARACTER
+ BNOPROGRAMVERSION         = CHARACTER
+ BNOSLTYPELENGTH           = NUMBER
+ BNOSLTYPESMAX             = NUMBER
+ BNOTRUEFALSECHR           = CHARACTER ARRAY(2)
+ INCLUDEPROGRAMINFO        = NUMBER
+ OPTIONTYPELOOP            = NUMBER
+ PARAMOPTION1              = CHARACTER
+ PARAMOPTION2              = CHARACTER
+ PARAMOPTION3              = CHARACTER
+ PARAMOPTION4              = CHARACTER
+ PARAMOPTION5              = CHARACTER
+ PARAMOPTION6              = CHARACTER
+ PARAMOPTIONINPUT          = CHARACTER ARRAY(6)
+ PARAMSHARETYPESBYOPTION   = NUMBER ARRAY(6,9999)
+ SHARETYPELOOP             = NUMBER
+ TARGETSHARETYPE           = NUMBER
+ TMPVAL                    = NUMBER
+ END [DEFINE]
 
 SETUP
+ INCLUDEPROGRAMINFO=TRUE
+ BNOLASTMODDATE='01/31/22'
```

Comment:
Recommending BNOLASTMODDATE be brought to present date so Banno Debug mode kicks in when CU's install this the first 60 days after it goes into baseline.

Reply:
DONE